core/src/main/java/jenkins/model/ModifiableTopLevelItemGroup.java

```diff
@@ -66,6 +66,7 @@
 
     /**
      * Creates a new job.
+     *
```

Comment:
Yes, otherwise this messes up Javadoc processing. I'm currently working on the extension indexer and single linebreaks are collapsed, resulting in such fun lines as `This does something. @author whoever @since whatever`.